src/dashboard/src/components/api/views.py

```diff
@@ -113,7 +113,10 @@ def get_unit_status(unit_uuid, unit_type):
     :return: Dict with status info.
     """
     ret = {}
-    job = models.Job.objects.filter(sipuuid=unit_uuid).filter(unittype=unit_type).order_by('-createdtime', '-createdtimedec')[0]
+    job = models.Job.objects.filter(
+        sipuuid=unit_uuid).filter(
+            unittype=unit_type).order_by(
+                '-createdtime', '-createdtimedec')[0]
```

Comment:
I'm assuming you're making this change to increase readability? I don't mind one way or the other. Would you be onboard if we try to agree on a rule and enforce it using a linter? Would that be possible in Python? I'd almost prefer to follow one style consistently even if it's not the more Pythonic way to do it.